src/tool/hpcrun/gpu/intel/opencl-api.h

```diff
@@ -0,0 +1,21 @@
+#include <CL/cl.h>
+
+#include "opencl-intercept.h"
+
+#ifndef _OPENCL_API_H_
+#define _OPENCL_API_H_
+typedef struct profilingData
+{
```

Comment:
use the convention of two space indents. see the files in the GPU subdirectory.

Reply:
done